test/api/test_config.py

```diff
@@ -0,0 +1,168 @@
+"""
+Copyright (c) 2020 Genome Research Limited
+
+Author: Aiden Neale <an12@sanger.ac.uk>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or (at
+your option) any later version.
+
+This program is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see https://www.gnu.org/licenses/
+"""
+
+import os
+import unittest
+from tempfile import TemporaryDirectory
+from core import typing as T, time
+from core.config import exception
+from api.config import Config, _validate, _schema
+
+
+class TestLoader(unittest.TestCase):
+    _tmp:TemporaryDirectory
+    _path:T.Path
+
+    def setUp(self) -> None:
+        self._tmp = TemporaryDirectory()
+        self._path = path = T.Path(self._tmp.name)
+
+        tmp_file = path / "config"
+        tmp_file.touch()
+
+
+    def tearDown(self) -> None:
+        self._tmp.cleanup()
+        del self._path
+
+
+
```

Comment:
One newline between class method and function definitions
Two newlines between code sections (not rigorously defined, but broadly grouped by similar functionality)